alkira/resource_alkira_probe_https.go

```diff
@@ -0,0 +1,207 @@
+package alkira
+
+import (
+	"context"
+
+	"github.com/alkiranet/alkira-client-go/alkira"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceAlkiraProbeHTTPS() *schema.Resource {
+	return &schema.Resource{
+		Description:   "Manage Alkira HTTPS Probe.",
+		CreateContext: resourceProbeHTTPSCreate,
+		ReadContext:   resourceProbeHTTPSRead,
+		UpdateContext: resourceProbeHTTPSUpdate,
+		DeleteContext: resourceProbeHTTPSDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Description: "The name of the HTTPS probe.",
+				Type:        schema.TypeString,
+				Required:    true,
+			},
+			"enabled": {
+				Description: "Whether the probe is enabled.",
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     true,
+			},
+			"network_entity": {
+				Description: "Network entity configuration.",
+				Type:        schema.TypeList,
+				Required:    true,
+				MaxItems:    1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"type": {
+							Description: "The type of network entity to probe." +
+								" Only `INTERNET_APPLICATION` supported for now.",
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"id": {
```

Comment:
Looks like we could totally avoid this block. So far it will be only `network_entity_id` and it could be only IFA ID.